src/clients/Deluge.ts

```diff
@@ -22,11 +22,22 @@ enum DelugeErrorCode {
 	RPC_FAIL = 4,
 	BAD_JSON = 5,
 }
-interface TorrentInfo {
-	complete: boolean;
-	save_path: string;
-	label?: string;
-}
+type injectData = [
```

Comment:
Types use `PascalCase`

```suggestion
type InjectData = [
```